### PR TITLE
Add wrappers for poretools 0.6.1a1

### PR DESCRIPTION
**Pull request by davebx**
Leaving this WIP for the moment, there is an issue in the tool code that breaks some of the tools.

### CODE REVIEW

**Comment by davebx** [comment]
Removed WIP because the bioconda package appears to be correct now

File: tools/poretools/poretools_extract.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        #if $group:
+            --group "$group"
+        #end if
+        #if $start:
+            --start "$start"
+        #end if
+        #if $end:
+            --end "$end"
+        #end if
+        "$input" > "$output"
```
Comment (bgruening) [inline tools/poretools/poretools_extract.xml]: single-quotes if possible

File: tools/poretools/poretools_hist.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<tool id="poretools_hist" name="Generate histogram" version="@VERSION@.0">
+    <description>of nanopore read lengths</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive">
+    <![CDATA[
+        poretools hist "$input" --saveas histogram.$extension --min-length $min_length
```
Comment (bgruening) [inline tools/poretools/poretools_hist.xml]: single-quotes :)

**Comment by bgruening** [comment]
Thanks Dave!

**Pull request merged**